src/main/java/uk/gov/hmcts/probate/model/ccd/raw/response/ResponseCaseDataParent.java

```diff
@@ -87,4 +87,5 @@ public class ResponseCaseDataParent {
     protected String solsNeedsPBAPayment;
     protected OrganisationPolicy applicantOrganisationPolicy;
     protected LocalDate lastEvidenceAddedDate;
+    protected String documentUploadedAfterCaseStopped;
```

Comment:
is this actually saved down to CCD?
If so needs to be in probate-commons, thus there would be a build.gradle change